/**:
  ros__parameters:
    min_distance: 0.2
    max_distance: 200.0
    start_angle: 0.0
    end_angle: 360.0  
    distance_unit: 0.25
    calibration_file: ""   
    scan_start_angle: 0.0
    scan_end_angle: 36000.0
    return_mode: 1
    config_vert: true     
    print_vert: true  
    config_vert_file: false
